db/blip_handler.go

```diff
@@ -49,6 +49,8 @@ const (
 
 type blipHandlerFunc func(*blipHandler, *blip.Message) error
 
+var ProposeChangesErr = base.HTTPErrorf(http.StatusConflict, "Use 'proposeChanges' instead")
```

Comment:
Follow typical error naming convention and be a bit more specific:

```suggestion
var ErrUseProposeChanges = base.HTTPErrorf(http.StatusConflict, "Use 'proposeChanges' instead")
```